pkg/scheduler/framework/plugins/noderesources/most_allocated_test.go

```diff
@@ -105,10 +106,15 @@ func TestNodeResourcesMostAllocated(t *testing.T) {
 			},
 		},
 	}
+	defaultResourceMostAllocatedSet := []config.ResourceSpec{
+		{Name: "cpu", Weight: 1},
+		{Name: "memory", Weight: 1},
```

Comment:
use v1.ResourceCPU and v1.ResourceMemory instead of "cpu" and "memory"?

Reply:
done